src/Components/test/testassets/Components.TestServer/RazorComponentEndpointsNoInteractivityStartup.cs

```diff
@@ -48,19 +48,37 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
         app.Map("/subdir", app =>
         {
-            if (!env.IsDevelopment())
+            app.Map("/reexecution", reexecutionApp =>
             {
-                app.UseExceptionHandler("/Error", createScopeForErrors: true);
-            }
```

Comment:
We seem to have removed from here. Is it intentional?
```
            if (!env.IsDevelopment())
            {
                app.UseExceptionHandler("/Error", createScopeForErrors: true);
            }
```

Reply:
No! Good catch